ssr-express/lib/AmpSsrUrlScheme.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const {URL} = require('whatwg-url');
+
+/**
+ * When applying server-side-render AMPs, the resulting pages need to be linked to
+ * AMP equivalents.
+ */
+class AmpSsrUrlScheme {
```

Comment:
Suggestion: rename to UrlMapping (as it has nothing to do with SSR or AMP per se)

Reply:
done.